src/test/resources/arquillian.xml

```diff
@@ -3,16 +3,24 @@
             xmlns="http://jboss.org/schema/arquillian"
             xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
 
-  <!--
-      <engine>
-          <property name="deploymentExportPath">target</property>
-      </engine>
-  -->
+  <engine>
+      <property name="deploymentExportPath">target</property>
+  </engine>
+
   <container qualifier="chameleon" default="true">
     <configuration>
       <property name="chameleonTarget">wildfly:9.0.0.Final:managed</property>
     </configuration>
   </container>
+  <container qualifier="tomcat7-managed">
+    <configuration>
+      <property name="chameleonTarget">${arq.container.chameleon.configuration.chameleonTarget}</property>
+      <property name="user">arquillian</property>
+      <property name="pass">arquillian</property>
+      <!-- relative to CATALINA_BASE/conf; catalinaBase is set by chameleon itself -->
+      <property name="serverConfig">../../../../../src/test/resources/tomcat7-server.xml</property>
```

Comment:
This folder is used for testing purposes only. Location goes up from `catalinaBase`. It's the limitation of current Tomcat Adapter which restricts the location of `server.xml` to be located within tomcat directory which is totally unnecessary.

See here: https://github.com/arquillian/arquillian-container-tomcat/blob/master/tomcat-managed-common/src/main/java/org/jboss/arquillian/container/tomcat/managed/TomcatManagedContainer.java#L154